pointmatcher/ICP.cpp

```diff
@@ -43,6 +43,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "MatchersImpl.h"
 #include "OutlierFiltersImpl.h"
 #include "ErrorMinimizersImpl.h"
+#include "PointToPlaneErrorMinimizer.h"
```

Comment:
The PointToPoint part of the job hasn't been done yet but I put the include for all the PointToPlane minimizers